README.md

```diff
@@ -26,7 +26,9 @@ we are gathered here today to get through this thing called LIFE."</em><strong>
   </div>
    </br>
 
-# e-wallet app
+# Budget app
+
+[Live Demo](https://leobudgetapp.herokuapp.com/)
```

Comment:
- [ ] I just got the first page and then it gave me the error, looks like I have to send the database or something you'll need.

![Screenshot from 2022-03-17 04-01-11](https://user-images.githubusercontent.com/22434660/158702179-d543691a-b286-4573-b697-3eada03268e1.png)

Reply:
in a real company, just a message helps and with this simple action, we're able to improve team communication.